server/app/controllers/applicant/ApplicantProgramReviewController.java

```diff
@@ -425,14 +426,19 @@ private Result renderIneligiblePage(
               .build();
       return ok(northStarApplicantIneligibleView.render(params)).as(Http.MimeTypes.HTML);
     } else {
-      return ok(
-          ineligibleBlockView.render(
-              request,
-              profile,
-              roApplicantProgramService,
-              messagesApi.preferred(request),
-              applicantId,
-              programDefinition));
+      try {
+        return ok(
+            ineligibleBlockView.render(
+                request,
+                profile,
+                roApplicantProgramService,
+                messagesApi.preferred(request),
+                applicantId,
+                programDefinition,
+                Optional.empty()));
```

Comment:
**suggestion:** let's add  `/* blockId= */ ` to make it clearer what this is.

Reply:
good call! added.